roxctl/central/initbundles/revoke.go

```diff
@@ -86,9 +140,20 @@ func revokeCommand(cliEnvironment environment.Environment) *cobra.Command {
 		Long:  "Revoke an init bundle for bootstrapping new StackRox secured clusters",
 		Args:  cobra.MinimumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return revokeInitBundles(cliEnvironment, args, flags.Timeout(cmd), flags.RetryTimeout(cmd))
+			var clusterNameOrIds []string
+			var force bool
```

Comment:
Hmm but this is a flag, so you should be able to simply do `cmd.Flags().GetBool("force")` instead of wrangling with the args.